From a86caa7a9beae39106635706c9ec428a2a53d9c7 Mon Sep 17 00:00:00 2001
From: Lu Jialin <lujialin4@huawei.com>
Date: Wed, 12 Jul 2023 10:25:12 +0800
Subject: [PATCH 12/36] [Huawei] PSI: enable CONFIG_PSI_FINE_GRAINED in
 openeuler_defconfig

Offering: HULK
HULK inclusion
category: feature
bugzilla: 188780

-------------------------------

enable CONFIG_PSI_FINE_GRAINED in openeuler_defconfig of x86 and arm64

Signed-off-by: Lu Jialin <lujialin4@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 +-
 arch/x86/configs/openeuler_defconfig   | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 87ebcead977c..8f76c3929942 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -92,7 +92,7 @@ CONFIG_TASK_IO_ACCOUNTING=y
 CONFIG_PSI=y
 #CONFIG_PSI_DEFAULT_DISABLED is not set
 CONFIG_PSI_CGROUP_V1=y
-
+CONFIG_PSI_FINE_GRAINED=y
 # end of CPU/Task time and stats accounting
 
 CONFIG_CPU_ISOLATION=y
diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 36ad3c6cad99..09f2162a864e 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -108,6 +108,7 @@ CONFIG_TASK_IO_ACCOUNTING=y
 CONFIG_PSI=y
 #CONFIG_PSI_DEFAULT_DISABLED is not set
 CONFIG_PSI_CGROUP_V1=y
+CONFIG_PSI_FINE_GRAINED=y
 # end of CPU/Task time and stats accounting
 
 CONFIG_CPU_ISOLATION=y
-- 
2.17.1

